From 139e8840e2b335068b77ca445a4bd460c32f412b Mon Sep 17 00:00:00 2001
From: "Hebbar, Gururaja" <gururaja.hebbar@ti.com>
Date: Mon, 23 Apr 2012 11:46:28 +0530
Subject: [PATCH 0941/1092] ARM: OMAP2+: am335x: use proper defines for
 Beaglebone

Use proper #defines for Beaglebone. Till now Low Cost EVM defines
were used for Beaglebone in some places.

Signed-off-by: Hebbar, Gururaja <gururaja.hebbar@ti.com>
---
 arch/arm/mach-omap2/board-am335xevm.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-omap2/board-am335xevm.c b/arch/arm/mach-omap2/board-am335xevm.c
index ed63b25..b31d5d4 100644
--- a/arch/arm/mach-omap2/board-am335xevm.c
+++ b/arch/arm/mach-omap2/board-am335xevm.c
@@ -1927,7 +1927,8 @@ static void setup_beaglebone_old(void)
 	/* Beagle Bone has Micro-SD slot which doesn't have Write Protect pin */
 	am335x_mmc[0].gpio_wp = -EINVAL;
 
-	_configure_device(LOW_COST_EVM, beaglebone_old_dev_cfg, PROFILE_NONE);
+	_configure_device(BEAGLE_BONE_OLD, beaglebone_old_dev_cfg,
+								PROFILE_NONE);
 
 	phy_register_fixup_for_uid(BBB_PHY_ID, BBB_PHY_MASK,
 					beaglebone_phy_fixup);
@@ -1944,7 +1945,7 @@ static void setup_beaglebone(void)
 	/* Beagle Bone has Micro-SD slot which doesn't have Write Protect pin */
 	am335x_mmc[0].gpio_wp = -EINVAL;
 
-	_configure_device(LOW_COST_EVM, beaglebone_dev_cfg, PROFILE_NONE);
+	_configure_device(BEAGLE_BONE_A3, beaglebone_dev_cfg, PROFILE_NONE);
 
 	/* TPS65217 regulator has full constraints */
 	regulator_has_full_constraints();
-- 
1.7.11.2

